forge/routes/api/projectActions.js

```diff
@@ -35,11 +35,13 @@ module.exports = async function (app) {
     }, async (request, reply) => {
         try {
             if (request.project.state === 'suspended') {
-                if (app.license.active() && app.billing) {
-                    if (!await app.billing.isProjectStartAllowed(request.project.Team, request.project)) {
-                        reply.code(402).send({ code: 'billing_required', error: 'Team billing not configured' })
-                        return
-                    }
+                try {
+                    // This will perform all checks needed to ensure this instance
```

Comment:
Re-reading these comments explaining what the function does is definitely a code smell, a clearer name should make them redundant!